docs/docs/aleph_zero/sdk.md

```diff
@@ -85,8 +99,10 @@ const account = keyring.addFromUri('//Alice')
 
 ### Initialize DEX and tokens
```

Comment:
Here, it should definitely be mentioned about `deploy` function and its relation with `load` function